source/archive/mattermost-limits.rst

```diff
@@ -1,3 +1,5 @@
+:nosearch:
```

Comment:
Is the intention for this page that we no longer want it returned in search results, but it is accessible by URL for now?

Reply:
Oh good question. We don't want it available at all.